apps/pie-docs/src/components/assistive-text/code/props.json

```
@@ -0,0 +1,26 @@
+{
+  "headings": [
+    "Prop",
+    "Options",
+    "Description",
+    "Default"
+  ],
+  "rows": [
+    [
+      "variant",
+      {
+        "type": "code",
+        "item": [
+          "string"
```

I think when there are only a few options we can list them out. (I probably wouldn't recommend committing this suggestion directly, I haven't tested it)
```suggestion
          "\"default\"",
          "\"error\"",
          "\"success\""
```